vue/src/modules/Auth/PasswordReset/ResetPasswordForm.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+  <div class="row">
+    <div class="col-md-4">
+      <div class="login-left">
+        <img src="/assets/img/apollo11-white.png" alt="" style="width: 80px"/>
+        <h3>{{ $t('Welcome') }}</h3>
+      </div>
+    </div>
+    <div class="login-right clearfix">
+      <div style="position: relative">
```

Comment:
@skavt It is better to use predefined bootstrap classes for such things. Instead of style we can write here: `class="position-relative" `